requirements.txt

```diff
@@ -0,0 +1,24 @@
+appnope==0.1.0
+decorator==4.1.2
+dj-database-url==0.4.2
+Django==1.11.4
+django-filter==1.0.4
+djangorestframework==3.6.3
+ipython==6.1.0
+ipython-genutils==0.2.0
+jedi==0.10.2
+Markdown==2.6.8
+olefile==0.44
+pexpect==4.2.1
+pickleshare==0.7.4
+Pillow==4.2.1
+prompt-toolkit==1.0.15
+psycopg2==2.7.3
+ptyprocess==0.5.2
+Pygments==2.2.0
+python-dateutil==2.6.1
+pytz==2017.2
+simplegeneric==0.8.1
+six==1.10.0
+traitlets==4.3.2
```

Comment:
It is better to maintain requirements manually instead of pip freeze for maintainability